johnsonandjohnson/Bodiless-JS
# fix(examples): Full RTE 'bold', 'italic', 'superscript' are reset

Full RTE: 'bold', 'italic', 'superscript' are reset after selecting 'strikethrough' style

## Test Instructions


1. open /richtext
2. in Full RTE section add some text
3. highlight some text
4. select 'bold', 'italic', 'superscript', 'underlined', 'link' -> all styles are applied to the highlighted text
5. select 'strikethrough'

Expected: all styles kept same as selected.

## Related Issues

Fixes: #228
